### PR TITLE
fix: refactor code input

### PR DESCRIPTION
**Pull request by chungnguyenhoang**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

**Review by huydosgtech — APPROVED** [review]

File: src/components/CodeInput/CodeInput.tsx [hunk 1]
```diff
@@ -182,11 +187,7 @@ const CellContainer = styled.View({
   justifyContent: 'space-between',
 })
 
-const ForwardRefTextInputComponent = forwardRef<TextInputRef, TextInputProps>((props, ref) => (
-  <TextInput {...props} ref={ref} />
-))
-
-const StyledTextInput = styled(ForwardRefTextInputComponent)(() => ({
+const StyledTextInput = styled.TextInput<{ref?: React.RefObject<TextInput>}>(() => ({
```
Comment (baonguyenhsts) [inline src/components/CodeInput/CodeInput.tsx:190]:
```suggestion
const ForwardRefTextInputComponent = forwardRef<TextInput, TextInputProps>((props, ref) => (
  <TextInput {...props} ref={ref} />
))

const StyledTextInput = styled(ForwardRefTextInputComponent)(() => ({
}))
```

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "update-test": "jest -u",
     "typecheck": "tsc --noEmit",
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
-    "prepack": "bob build",
+    "prepare": "bob build",
```
Comment (baonguyenhsts) [inline package.json:20]: rollback this change

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

**Review by ledutu2 — APPROVED** [review]

**Review by baonguyenhsts — APPROVED** [review]

**Review by honghoangsts — APPROVED** [review]
